.eslintrc.js

```diff
@@ -5,10 +5,11 @@ module.exports = {
     "es6": true,
     "node": true
   },
-  "plugins": ["jest"],
+  "plugins": ["jest", "prettier"],
```

Comment:
Can you remove eslint-plugin-prettier, please? See https://github.com/prettier/prettier/issues/7828

Reply:
Also, this ESLint config is kinda ok: https://github.com/lydell/js-tokens/blob/9992be622597981b00d711481036c01687300b3d/.eslintrc.js

It automatically enforces som ES6 syntax.

Reply:
I took it but slightly adjusted.

`prefer-destructuring` for variable assignment is *ugly*, IMHO.
Also `jest/no-try-expect` is relaxed to `warn`. Jest's `expect(..).toThrow(..)` doesn't allow to make multiple matches (or I do not know how to do it :) )